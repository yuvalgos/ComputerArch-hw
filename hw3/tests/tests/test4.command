./dflow_calc tests/test4.dat tests/test4.in d41 d88 d150 p83 p121 d21 d76 p175 p106 d98 d182 d127 d177 p87 d196 p151 p105 p74 p129 p125 p113 p92 d171 d38 d23 p180 d23 p141 d19 p147 p108 p128 d103 d76 d129 d13 p116 d101 d67 p188 p50 p135 p140 d48 d129 p10 p185 d123 d82 d164 p195 p107