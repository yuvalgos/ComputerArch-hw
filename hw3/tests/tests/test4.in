8 21 30 23
3 22 28 26
3 4 17 20
0 15 23 18
3 3 4 22
8 12 10 18
2 17 10 3
6 13 4 29
7 20 10 6
3 31 8 31
6 1 9 7
5 22 22 7
8 9 24 4
8 6 25 1
3 15 26 21
5 20 14 20
0 12 6 31
7 30 21 14
3 7 21 31
1 2 27 23
4 24 19 18
2 9 30 15
2 21 20 15
0 3 22 22
3 28 5 30
8 24 25 3
0 4 24 13
6 19 11 16
4 20 18 12
0 4 24 7
5 14 29 9
4 10 4 28
8 26 17 3
8 4 15 30
7 10 8 27
2 24 20 12
1 19 29 0
5 7 22 16
6 31 25 22
7 17 5 4
2 23 11 2
3 12 3 21
1 31 26 13
6 15 22 25
3 28 0 16
4 8 30 23
6 7 13 6
8 23 17 11
6 5 20 9
3 22 6 21
7 23 24 3
0 4 20 15
0 12 20 23
6 1 20 3
0 20 6 6
1 11 10 25
2 1 2 31
0 6 15 11
6 11 21 25
2 9 23 1
5 1 0 15
3 9 8 10
4 18 19 10
6 31 2 1
7 3 6 29
7 14 1 27
0 7 20 11
0 15 10 0
7 26 0 4
4 28 13 15
7 7 25 22
5 0 31 19
5 3 4 25
7 1 30 6
5 12 28 8
6 3 5 24
7 19 11 28
3 23 4 20
7 16 12 13
2 13 11 14
3 24 20 2
8 0 21 29
4 27 26 14
6 7 20 5
4 2 7 24
8 1 15 8
7 16 10 13
4 11 8 23
5 4 22 22
2 1 0 0
4 26 0 26
7 0 0 9
6 3 23 0
6 7 17 1
8 27 24 27
7 21 25 26
1 5 11 12
2 16 30 19
0 11 19 30
5 1 10 4
0 28 17 17
1 8 13 23
7 24 24 24
5 15 18 2
2 24 29 10
4 10 0 11
2 24 27 15
2 31 23 28
1 25 8 13
8 6 25 10
4 11 22 3
3 8 17 4
2 23 18 19
1 6 17 22
1 5 23 10
5 26 19 31
2 18 3 3
7 16 25 8
8 30 4 31
1 14 1 27
7 24 16 13
3 17 30 20
7 29 30 15
3 13 26 15
1 18 4 26
8 0 22 14
7 27 2 21
4 19 21 29
3 31 7 24
7 14 14 6
8 26 4 16
3 27 20 30
5 26 9 6
0 19 2 19
1 13 26 29
1 11 15 15
6 8 29 24
4 5 13 28
0 22 20 24
5 12 17 7
1 6 14 20
5 13 23 20
4 17 22 0
8 31 23 20
2 6 25 12
5 13 2 5
3 28 13 11
2 16 2 25
5 5 2 28
5 30 13 21
8 12 4 14
1 17 6 7
6 24 2 17
7 13 17 12
4 30 5 14
0 27 27 26
4 0 20 15
5 16 6 12
5 13 29 12
4 7 2 27
4 31 13 22
1 21 10 19
5 6 28 24
3 15 26 11
4 30 15 31
3 30 4 22
8 24 22 12
2 8 31 2
4 2 30 6
5 2 6 31
1 25 10 30
6 17 1 18
5 8 18 21
8 31 19 22
5 17 29 7
7 14 24 21
8 22 21 30
8 25 21 6
2 2 20 23
2 4 15 15
4 16 4 10
2 1 16 0
3 16 30 11
7 4 30 16
5 27 9 1
3 24 12 9
0 29 8 12
6 6 29 11
1 21 30 5
0 26 18 26
4 24 28 29
3 5 20 29
1 20 29 3
6 21 23 20
3 0 5 21
7 18 14 13
3 18 26 2
8 23 26 18
7 2 28 5
7 23 20 5
