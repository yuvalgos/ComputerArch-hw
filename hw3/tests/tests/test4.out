Reading the opcodes latency info from tests/test4.dat ... Got latency for 9 opcodes
Reading the program file tests/test4.in ... Found 200 instructions
getProgDepth()==214
getInstDeps(41)=={23,22}
getInstDeps(88)=={49,49}
getInstDeps(150)=={88,129}
getDepDepth(83)==70
getDepDepth(121)==114
getInstDeps(21)=={17,14}
getInstDeps(76)=={58,69}
getDepDepth(175)==176
getDepDepth(106)==111
getInstDeps(98)=={76,17}
getInstDeps(182)=={165,135}
getInstDeps(127)=={95,122}
getInstDeps(177)=={161,162}
getDepDepth(87)==62
getInstDeps(196)=={189,178}
getDepDepth(151)==139
getDepDepth(105)==79
getDepDepth(74)==53
getDepDepth(129)==111
getDepDepth(125)==111
getDepDepth(113)==84
getDepDepth(92)==79
getInstDeps(171)=={99,124}
getInstDeps(38)=={-1,11}
getInstDeps(23)=={11,11}
getDepDepth(180)==126
getInstDeps(23)=={11,11}
getDepDepth(141)==78
getInstDeps(19)=={-1,-1}
getDepDepth(147)==98
getDepDepth(108)==77
getDepDepth(128)==129
getInstDeps(103)=={62,84}
getInstDeps(76)=={58,69}
getInstDeps(129)=={119,113}
getInstDeps(13)=={-1,10}
getDepDepth(116)==96
getInstDeps(101)=={79,77}
getInstDeps(67)=={34,-1}
getDepDepth(188)==180
getDepDepth(50)==58
getDepDepth(135)==111
getDepDepth(140)==130
getInstDeps(48)=={28,21}
getInstDeps(129)=={119,113}
getDepDepth(10)==0
getDepDepth(185)==153
getInstDeps(123)=={115,103}
getInstDeps(82)=={68,65}
getInstDeps(164)=={163,160}
getDepDepth(195)==195
getDepDepth(107)==86
